lib/cuse_lowlevel.c

```diff
@@ -319,6 +319,9 @@ struct fuse_session *cuse_lowlevel_setup(int argc, char *argv[],
 	if (res == -1)
 		goto err_sig;
 
+	/* Disable the receiving and processing of FUSE_INTERRUPT requests */
+	fuse_session_disable_interrupts(se);
+
```

Comment:
This would entirely disable for cuse? I'm not sure if this is right. Maybe add a flag  `CUSE_NO_INTERRUPT` (see `CUSE_UNRESTRICTED_IOCTL`) and then set by `struct cuse_info::flags`?